app/controllers/repairs_controller.rb

```diff
@@ -24,6 +25,7 @@ def edit
 
   def update
     @repairs = Repair.find(params[:id])
+    @repairs.owner_id = current_user.owner.id
```

Comment:
This line is not really necessary.  The owner_id is set when the repair is created, so it does not need to be set again.  `@repairs` is a clumsy name, should be `@repair' because you are just dealing with one record

Reply:
meant to say '@repair' not `@Repair'

Reply:
Ahh, anyway, it should not be capitalized